### PR TITLE
Log string errors as is in onConnectionError

### LINKED ISSUE #54: Consider `data` parameter from connectionError callback
Currently, `palindrom-polymer-client` constructs the error string on its own and doesn't consider `data` parameter which could contain useful information.

### PR DESCRIPTION
**Pull request by alshakero**
Fixes #54.

He is the current output:
image

### CODE REVIEW

File: palindrom-client.html [hunk 1]
```diff
@@ -618,13 +625,15 @@
         </style>
         <div class="box-container" hidden$="{{!showReconnectingIn}}">
             <div class="box notice tappable" on-tap="reconnectNow">
-                <strong><span class="hourglass">⏳</span> Not connected.</strong> Connecting in {{reconnectionSeconds}}s...
```
Comment (tomalec) [inline palindrom-client.html]: Are those whitespaces needed? it seems like broken identation

**Comment by alshakero** [comment]
This one? 
image

**Comment by tomalec** [comment]
> This one? 

Nope, this one: https://github.com/Palindrom/palindrom-polymer-client/pull/55/files#diff-98ae23f53c566a6d2ea602564c74bb56R640

**Comment by alshakero** [comment]
Update this PR. Current output is:
How errors look now:

### Restarting the app:

connection error 404

### Killing the server
connection server restart

### Sending an add patch:
connection server error


### Too big number:
image

**Comment by warpech** [comment]
In case of "Sending an add patch", why does it display two stack traces, not one?

**Comment by alshakero** [comment]
> In case of "Sending an add patch", why does it display two stack traces, not one?

Because they're indeed two seperate (asyncronously seperate) stack traces. One comes in a raw non-JSON message and another comes as `CloseEvent` later.

**Comment by alshakero** [comment]
I can omit the second one though.

**Comment by alshakero** [comment]
Is this good to go?

**Comment by tomalec** [comment]
LGTM

**Pull request merged**